extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json

```diff
@@ -21,15 +21,60 @@
           "title": "Feedback Status",
           "type": "string",
           "description": "Provide status of a delivery attempt.",
-          "enum": ["error", "exclude", "send", "bounce", "delay"],
+          "enum": ["error", "exclude", "send", "bounce", "delay", "duplicate", "blocklisted"],
           "meta:enum": {
             "error": "Mesage processing failed.",
             "exclude": "Messageprofile is filtered out for sending the message",
             "send": "Messages such as Email, SMS, Push that have been dispatched to a person's account, address, or device.",
             "bounce": "Message failed to deliver, and no longer will be retried.",
-            "delay": "message failed to deliver, but will be retry after next schedule time."
+            "delay": "message failed to deliver, but will be retry after next schedule time.",
+            "duplicate": "message is skipped since it's a duplicate for this groupID/pushtoken pairing "
```

Comment:
blocklisted description is missing

Reply:
Can we have some UI/User friendly neta:enums for the newly added and existing enums ? The meta:enums surface on the UI in the drop downs so we tend to keep them as short as possible. 

Suggested values are 

"error": "Message processing failed",
"exclude": "Message profile filtered out",
"send": "Sent messages",
"bounce": "Message bounced",
"delay": "Message delayed"
"duplicate": "Message duplicated"